src/PowerBalancerAgent.cpp

```diff
@@ -143,23 +146,29 @@ namespace geopm
                                            double max_power,
                                            double time_window,
                                            bool is_single_node,
-                                           int num_node)
+                                           int num_node,
+                                           const std::set<int> &active_core)
         : Role(num_node)
         , m_platform_io(platform_io)
         , m_platform_topo(platform_topo)
         , m_sample_agg(sample_agg)
         , m_num_domain(m_platform_topo.num_domain(GEOPM_DOMAIN_PACKAGE))
```

Comment:
In principle the power control domain could be something other than the package, but the name of the control were are pushing has "PACKAGE" in the name, so introducing that ambiguity doesn't really make much sense.

Reply:
It's confusing when you consider the next line ```m_num_core```, is all.  I'd rather we were consistent when we setup for loops over domains so that you don't have to do sleuthing to ensure the loop is iterating over what you expect it to, especially now since we're considering multiple domains (PACKAGE and CORE).

To your point, this could have been ```m_num_default_power_domain``` or similar to make it more clear.

Reply:
Right on, I'll rename the member.